ast/capabilities.go

```diff
@@ -37,7 +32,9 @@ func CapabilitiesForThisVersion() *Capabilities {
 
 // LoadCapabilitiesJSON loads a JSON serialized capabilities structure from the reader r.
 func LoadCapabilitiesJSON(r io.Reader) (*Capabilities, error) {
-	d := util.NewJSONDecoder(r)
-	var c Capabilities
-	return &c, d.Decode(&c)
+	caps, err := capabilities.LoadCapabilitiesJSON(r)
+	return (*Capabilities)(caps), err
 }
+
+// Capabilities is an alias for capabilities.Capabilities (for backwards compatibility).
+type Capabilities capabilities.Capabilities
```

Comment:
👈 I don't think these changes achieve what's declared here. If you had been using the `ast.Capabilities` struct before, you're in trouble when this change merges.